source/databases-collections.txt

```diff
@@ -96,6 +96,13 @@ its type. Index key types include:
 - Text
 - Hashed
 
+.. note::
```

Comment:
I agree with not needing to make this own page, however as part of these updates can we pull the Indexes content into its own heading on this page? Right now it's nested under `View Collection Documents and Schema` which isn't necessarily where I'd think to look for info on indexes.

Maybe under "databases and collections" add a new section called Indexes with all the info specific to Indexes.

Reply:
Agreed, it's a little hidden. Made small tweaks:

- Bumped the heading up a level so it appears on the ToC.
- Moved the intro line "Click a collection to view its documents, schema, and indexes." above **View Collection Documents and Schema** to introduce that section and the **Indexes** section, now on the same level.

Other than that no content's moved since the page is in the order of the Database dropdown options in the VSCE

Reply:
Do you think the section should be **View and Create Indexes** ?